codetransformer/utils/immutable.py

```diff
@@ -145,15 +149,21 @@ def _create_init(slots, defaults):
         return _no_arg_init, ()
 
     ns = {'__initialize_slot': initialize_slot}
-    slotnames = tuple(s.strip('*') for s in slots)
+    # filter out lone star
+    slotnames = tuple(filter(bool, (s.strip('*') for s in slots)))
```

Comment:
`filter(None, it)` has a fast path in C code, so I'd recommend using that. Also it's more Pythonic ;)